package.json

```diff
@@ -10,11 +10,15 @@
     "test": "jest --watchAll"
   },
   "dependencies": {
+    "@auth/mongodb-adapter": "^1.0.0",
     "@next/font": "^13.4.19",
     "eslint": "8.44.0",
     "eslint-config-next": "13.4.8",
+    "mongodb": "^5.8.1",
```

Comment:
I'm not sure why you would need both `mongodb` and `@auth/mongodb-adapter`. It's possible to integrate authentication without these packages.

But if you managed to implement authentication, and it looks like you did, you can leave them for now. Maybe come back later and have another look to check if you actually need them.